### ISSUE TITLE
Some problems with the i18n text

### ISSUE THREAD

**Issue by wordpure** [sixcious/infy-scroll#65]
### Problem

1. i18n field of some text changed

	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/48ac9e2c-ece4-47dc-8a93-fbb906e7f0da)
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/555043ad-da5f-43c8-b4c7-f6464c290ecb)

2. Some texts do not have i18n counterparts 

	<details>
	<summary>Images</summary>
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/e5d93c38-1d3c-432f-ac43-79db2f8eb605)
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/b3db6147-1607-4303-b23b-913688168343)
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/96259693-90bc-4dc1-984a-ad291bb41241)
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/9cf0c41a-983e-469e-b616-30cc012875cd)
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/2e7324c2-a2fb-40c6-a85d-33e9a3017a82)
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/0016bca0-ad29-49a5-b2a7-0b7eef114c68)
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/6abda783-c60b-4e3f-8135-656e81561217)

	</details>

3. Text that is not in i18n but will automatically change according to the user's language

	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/7726f6ee-a01c-4322-9cc1-810cc7df44d2)
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/69af2d8f-4816-4bd1-8b9f-d5ed3f2ef54f)

4. Some i18n don't seem to show up in the interface
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/ad60ab4a-d89b-4a08-bc91-de8aaedf3e8b)
	
	![图片](https://github.com/sixcious/infy-scroll/assets/68013962/54a8109e-69c3-444b-bec4-d66ff5c26b40)

### URL

irrelevant

### Version

0.8

### Browser

Firefox 114

### OS

Windows 11

### Device

PC

**Comment by sixcious** [comment]
Hi again wordpure, thank you for opening this issue (and sorry for the delay)!

Yes, so I think you might have been using your own GitHub translation with the Web Store build, but the GitHub build (April 2023) is about a month behind the Web Store build (May 2023). I added and changed some things, so it won't sync up perfectly. You brought up some other interesting things I didn't notice though. For example, I never i18n'd the Keyboard Shortcuts so they aren't translatable; I'll do that for the next release. (I'm not sure what's wrong with Point 3 though; isn't that a good thing?)

Unfortunately, my local copy of Infy isn't quite ready to be pushed up to GitHub. I'm not sure when I'll be working on Infy again, but we can keep this issue open for now.

Thank you again for your translation work!💜

**Comment by wordpure** [comment]
> Yes, so I think you might have been using your own GitHub translation with the Web Store build, but the GitHub build (April 2023) is about a month behind the Web Store build (May 2023).

Thank you for your reminder. 

As you said, the translation issue was due to differences in the extension store and Github versions. I deleted the extension downloaded by Firefox Addons (if not deleted, just disable the extension store version, and then temporarily install the Github version, it seems that the new i18n text of the Github version will not work due to the same version number), after adding the Github extension, Most of the translated text displayed normally.

> I'm not sure what's wrong with Point 3 though; isn't that a good thing?

It's a good thing indeed, I'm just curious as to why it happened. But now it seems that the browser should automatically add the prompt text for the number type input with max, min and step set.

> my local copy of Infy isn't quite ready to be pushed up to GitHub. I'm not sure when I'll be working on Infy again, but we can keep this issue open for now.

Thank you for your great work and I look forward to seeing Infy get better and more perfect.
